src/style/fog.js

```diff
@@ -26,6 +26,9 @@ const properties: Properties<Props> = new Properties({
 
 const TRANSITION_SUFFIX = '-transition';
 
+export const FOG_PITCH_START = 55;
+export const FOG_PITCH_END = 65;
```

Comment:
Seems totally plausible that these aren't worth making inputs. Making the FOV adjustable might change that, but probably only very slightly so that this seems totally reasonable 👍